plugins/http/src/commands.rs

```diff
@@ -29,6 +87,7 @@ pub async fn fetch<R: Runtime>(
     data: Option<Vec<u8>>,
     connect_timeout: Option<u64>,
     max_redirections: Option<usize>,
+    proxy: Option<ProxyConfig>,
```

Comment:
Let's combine these options into a new struct `ClientConifg` and remove `#[allow(clippy::too_many_arguments)]`